README.md

```diff
@@ -35,24 +35,30 @@ The `RegisterClientManager` is the entry point of the Registers Ruby client:
 require 'register_client_manager'
 
 registers_client = RegistersClient::RegisterClientManager.new({
+  api_key: "68fc3741-fab0-4615-a9c8-08a8696d4371",
```

Comment:
I wonder if you should change this to 'YOUR-API-KEY-HERE' or something, so it is obvious not to use this example key?